prismic-model/js/exhibitions.js

```diff
@@ -32,8 +32,34 @@ const Exhibitions = {
   Promo: {
     promo
   },
-  // When we get a better relational model up, this will be deprecated
-  'Related promos': {
+  Resources: {
+    resources: list('Resources', {
+      resource: link('Resource', 'document', ['exhibition-resources'])
+    })
+  },
+  Related: {
+    relatedOfs: list('Related of', {
```

Comment:
The language of "related" is also asking content editors to put exactly that in, where as what we're modelling is what we have, not whether someone thinks things are similar.